201

```diff
@@ -0,0 +1,373 @@
+"""
```

Comment:
This looks like a slightly different version than `statsmodels/kernel_methods/bandwidths.py`. Ok to delete this one?

```
| $ diff 201 statsmodels/kernel_methods/bandwidths.py
140c140
< def _botev_fixed_point(t, M, I, a2):
---
> def _botev_fixed_point(t, M, J, a2):
142c142
<     I = large_float(I)
---
>     J = large_float(J)
145c145
<     f = 2 * np.pi**(2 * n) * np.sum(I**n * a2 * np.exp(-I * np.pi**2 * t))
---
>     f = 2 * np.pi**(2 * n) * np.sum(J**n * a2 * np.exp(-J * np.pi**2 * t))
151c151
<             np.sum(I ** s * a2 * np.exp(-I * np.pi ** 2 * time))
---
>             np.sum(J ** s * a2 * np.exp(-J * np.pi ** 2 * time))
176c176
<         data = model.exog
---
>         data = np.squeeze(model.exog)
201c201
<         I = np.arange(1, N, dtype=int)**2
---
>         J = np.arange(1, N, dtype=int)**2
209c209
<                                      args=(M, I, SqDCTData))
---
>                                      args=(M, J, SqDCTData))
```